Libraries/src/Amazon.Lambda.Serialization.SystemTextJson/Converters/DateTimeConverter.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Amazon.Lambda.Serialization.SystemTextJson.Converters
+{
+    /// <summary>
+    /// DateTime converter that handles the JSON read for deserialization might use an epoch time.
+    /// </summary>
+    public class DateTimeConverter : JsonConverter<DateTime>
+    {
+        private static readonly DateTime EPOCH_DATETIME = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
You can use `DateTime.UnixEpoch` instead.

Reply:
Nice, I didn't realize `DateTime.UnixEpoch` was added.